File: notification-service/Functions/Infrastructure/HealthCheck.cs
using System;
using System.IO;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Mvc;
using Microsoft.Azure.WebJobs;
using Microsoft.Azure.WebJobs.Extensions.Http;
using Microsoft.AspNetCore.Http;
using Microsoft.Extensions.Logging;
using Newtonsoft.Json;

namespace Serverless
{
    public class HealthCheck
    {
        [FunctionName("Ping")]
        public async Task<IActionResult> Run(
            [HttpTrigger(AuthorizationLevel.Anonymous, "GET", Route = "api/ping")]
            HttpRequest req,
            ILogger log)
        {
            log.LogInformation("C# Ping trigger function processed a request.");

            return new OkObjectResult("OK");
        }
    }
}

File: shipping-service-java/src/main/java/com/thinktecture/serverless/functions/HealthCheck.java
package com.thinktecture.serverless.functions;

import java.util.*;
import com.microsoft.azure.functions.annotation.*;
import com.microsoft.azure.functions.*;

public class HealthCheck {
    @FunctionName("Ping")
    public HttpResponseMessage run(
        @HttpTrigger(name = "req", route = "ping", methods = { HttpMethod.GET }, authLevel = AuthorizationLevel.ANONYMOUS)
        HttpRequestMessage<Optional<String>> request,
        final ExecutionContext context) {
        context.getLogger().info("Ping HTTP trigger processed a request.");

        return request.createResponseBuilder(HttpStatus.OK).body("OK").build();
    }
}
